wiki/FAQ/en.md

```diff
@@ -56,161 +77,225 @@ _[Osu! Tatakae! Ouendan][ouendan wikipedia]_ is a rhythm game for the Nintendo D
+
+When converting an osu!standard beatmap, the default key range would be around 4 to 7 keys. With the [xK][xK wikilink] game modifier, you can manually set the number of keys you play with 1 to a total of 9 keys with a score multiplier reduction. However, this game modifier will not work with osu!mania-specific beatmaps. With the [Co-Op][Co-Op wikilink] game modifier, you can split the stage in two. This will use Co-Op's control scheme and will allows you to play with 2 to a total of 18 keys with a score multiplier reduction. Do note that while the preset number of keys will not be doubled on osu!mania-specific beatmaps, it will split the stage in two (left stage priority if odd-numbered), use Co-Op's control scheme, and no score multiplier reduction.
+
+#### Basic explanation
+##### Playfield
+![osu!mania playfield][osu!mania playfield image]
+
+By default, the flow of the hitobjects falls from top to bottom of the convenyor (the starting arrow will show where hitobjects will flow to), with the key control at the bottom and the judgement barline above the key control. To reverse the control and flow of the playfield, it can be changed in the Options sidebar; click on _osu!mania layout_ to set osu!mania keys settings and enable _Vertically flip playfield (DDR style)_ for this key setting manually.
+
+Healthbar is located at the right side of the conveyor belt. Please note that health drain is disabled in osu!mania, so only the hitobjects will affect the healthbar. Combo will not break when playing the keys on empty judgement barline.
+
+##### Tapbars
+![osu!mania's tapbars][osu!mania tapbars image]
+
+Tapbars are the hitcircles of osu!mania. Tapbars are falling bars to be tapped on the judgement barline, with correct key corresponding to each of the tapbar falls into. More correct keys must be tapped simultaneously if the tapbars fall simultaneously.
+
+##### Holdbars
+![osu!mania's Holdbars][osu!mania Holdbars image]
 
-This mode has been widely used in almost all of the major rhythm games that require good hand/leg coordination, where the beat notes with their quantity depending on BPM and difficulty move on the conveyer, and the player have to press the correct key for that specific beat note in time. Though the game mode was originally made as an imitation of Beatmania style of playing, however, by changing the number of keys, you can make it look more like Guitar Hero (5) or Dance Dance Revolution (4) and so on.
+Holdbars are the sliders and spinners of osu!mania. When the holdbar reached the judgement barline, tap the starting tapbar in time with correct key, hold, and release at the ending tapbar of the holdbar.
```

Comment:
mix of past and present tense?
second sentence:
`When the holdbar reaches the judgement barline, tap the starting tapbar in time with correct key, hold, and release it at the ending tapbar of the holdbar.`